src/wp-includes/global-styles-and-settings.php

```diff
@@ -320,10 +321,13 @@ function wp_add_global_styles_for_blocks() {
 			)
 		);
 
-		$cache_key = "wp_styles_for_blocks:$hash";
-		$cached    = get_site_transient( $cache_key );
-		if ( ! is_array( $cached ) ) {
-			$cached = array();
+		$cache_key = 'wp_styles_for_blocks';
+		$cached    = get_transient( $cache_key );
```

Comment:
Why is this changed to `get_transient ` from `get_site_transient`?

Reply:
This cache needs to be site specific, because it uses global settings and the computed styles for blocks in the hash, which incorporates users settings. When the hash key is part of the cache key name, every site on a multisite network with different settings will generate unique keys, which will expire naturally in 10 minutes.

To avoid the extra DB lookup for a transient with an expiration, I'm looking at making the cache key consistent, and put the hash inside the cached data for invalidation, but that means we need to save separate transients for each site on a multisite install, rather than as a shared transient for the whole network.